test/necb/unit_tests/expected_results/necb2011_compliance_unitary_efficiencies_expected_results.csv

```diff
@@ -1,9 +1,9 @@
 Heating Type,Min Capacity (Btu per hr),Max Capacity (Btu per hr),Seasonal Energy Efficiency Ratio (SEER),Energy Efficiency Ratio (EER)
-Electric Resistance,0,65000,14.0,
-Electric Resistance,65001,249999,,9.7
-Electric Resistance,250000,759999,,8.39
-Electric Resistance,760000,9999999999,,8.12
-All Other,0,65000,14.0,
-All Other,65001,249999,,9.7
-All Other,250000,759999,,8.21
-All Other,760000,9999999999,,7.94
```

Comment:
@lymereJ The input NECB reference values are entered in a json file. In this case these values are in file './necb/NECB2011/data/unitary.json'. If possible we want to keep the same values here in the test expected results file as the ones in the json file on the input side. When we convert the input SEER values in standards we use the method that accounts for the effect of the fan ('seer_to_cop' now called). When we are checking an osm file for the correct values of COP, we need to convert this back to SEER. At this stage the COP does not include the effect of the fan anymore. In the test it seems we want to use the method 'cop_to_seer_no_fan' so we can keep the original values of SEER values in the expected results file?

Reply:
> The input NECB reference values are entered in a json file. In this case these values are in file './necb/NECB2011/data/unitary.json'. If possible we want to keep the same values here in the test expected results file as the ones in the json file on the input side. 

As it stands, this PR doesn't touch the efficiency JSON files, or the expected results. The change you commented on is "outdated" (as per GitHub), I think it comes from a previous commit.

> When we convert the input SEER values in standards we use the method that accounts for the effect of the fan ('seer_to_cop' now called). When we are checking an osm file for the correct values of COP, we need to convert this back to SEER. At this stage the COP does not include the effect of the fan anymore. In the test it seems we want to use the method 'cop_to_seer_no_fan' so we can keep the original values of SEER values in the expected results file?

I'm not sure I understand. As it stands, this PR takes the requirements and convert them for input to an OpenStudio model as gross COP values (impact of the fan on the efficiency rating has been removed) which I believe is what EnergyPlus expects. The only changes that were made to the NECB tests is to convert the gross COP values ("no fan") from the test models to either EER (`cop_no_fan_to_eer`) or SEER (`cop_no_fan_to_seer) and compare them to the expected results which match what's in `./necb/NECB2011/data/unitary.json`. Does that match your expectations?

In the US, as per AHRI testing standards, rated SEER and EER are always "net" efficiencies, not "gross", meaning that they always include the impact of fan power.

Reply:
@lymereJ  I must not have looked at the latest code. Yes it seems all is good then. It makes it easier when the inputs to the jsons match the test expected values in the tests. Thanks